source/extensions/filters/network/dubbo_proxy/decoder.h

```diff
@@ -13,37 +14,129 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace DubboProxy {
 
-class DecoderCallbacks {
+#define ALL_PROTOCOL_STATES(FUNCTION)                                                              \
+  FUNCTION(StopIteration)                                                                          \
```

Comment:
The style here seems inconsistent (PascalCase vs camelCase), what is the motivation for that?

Reply:
sorry, it should use PascalCase, i have modified it.